sigstore/_internal/tuf.py

```diff
@@ -197,7 +254,21 @@ def get_rekor_keys(self) -> list[bytes]:
 
         May download files from the remote repository.
         """
-        keys = self._get("Rekor", ["Active"])
+        keys = []
+
+        self._trusted_root = self._get_trusted_root()
+        if self._trusted_root:
+            for key in self._trusted_root.tlogs:
+                if not _is_timerange_valid(
+                    key.public_key.valid_for, allow_expired=False
+                ):
+                    continue
```

Comment:
If we're ignoring keys because they're expired, is this something that we want to log about? I feel like it could be confusing for a user if they've been supplying a trusted root and the keys stop working suddenly without logging anything.

Reply:
Yeah, that might be weird from a UX perspective. If we do this, should we also log for this in `_get`, the non bundled case?

Reply:
I think it's fine. If this happens will get a `CTFE keys not found in TUF metadata` error and will report the issue and it should be obvious what the problem is.